examples/derive-patterns/join-one-sided.flow.yaml

```diff
@@ -9,6 +9,42 @@ collections:
       required: [Key]
     key: [/Key]
 
+    derive:
+      using:
+        sqlite:
+          migrations:
+            - |
+              create table join_state (
```

Comment:
nit: these are potentially close enough that you could reference a common migration SQL?